tor/torrc

```diff
@@ -0,0 +1,192 @@
+## Configuration file for a typical Tor user
```

Comment:
Same here, if this is the default, why do we have to provide it?

Reply:
Oh this is not the default any more I modified it. We can actually just have a file (or comment in code) on the hidden service config parts. This is the only bits I added and that users need to add
```
HiddenServiceDir /var/lib/tor/hidden_service/
HiddenServicePort 9939 127.0.0.1:9939
```

Reply:
My intention with adding these files was to make running Tor as easy as possible in the 'typical' case since you expressed before that we want zero config.

Reply:
Remove this file and added a section to the main README describing how to use Tor.

Reply:
Ah okay, now I get it!

Yes only the relevant part is way clearer I think, thank you!